clio-server/src/main/scala/org/broadinstitute/clio/server/service/ReadGroupService.scala

```diff
@@ -49,23 +79,40 @@ object ReadGroupService {
   }
 
   private[service] val ConverterV1Key =
-    SameFieldsTypeConverter[TransferReadGroupV1Key, ModelReadGroupKey]
+    CaseClassTypeConverter[TransferReadGroupV1Key, ModelReadGroupKey] {
+      _.updated("location", "Unknown")
+    }
 
   private[service] val ConverterV1Metadata =
     SameFieldsTypeConverter[
       TransferReadGroupV1Metadata,
       ModelReadGroupMetadata
     ]
 
+  private[service] val ConverterV2Key =
+    SameFieldsTypeConverter[TransferReadGroupV2Key, ModelReadGroupKey]
+
   private[service] val ConverterV1QueryInput =
-    SameFieldsTypeConverter[
+    CaseClassTypeConverter[
       TransferReadGroupV1QueryInput,
       ModelReadGroupQueryInput
-    ]
+    ] { _.filterKeys(_ != "location") }
 
   private[service] val ConverterV1QueryOutput =
-    SameFieldsTypeConverter[
+    CaseClassTypeConverter[
       ModelReadGroupQueryOutput,
       TransferReadGroupV1QueryOutput
+    ] { _.filterKeys(_ != "location") }
```

Comment:
So this removes `location` from the output for V1? The code behavior is not immediately obvious to me.